frontend/src/components/page-watch/index/sermons.js

```diff
@@ -21,11 +24,23 @@ const Sermons = () => {
     }
   `);
   return (
-    <div>
+   <div className="grid grid-cols-2 lg:grid-cols-3 gap-x-5 gap-y-8 py-36 max-w-container px-4">
       {data.allStrapiSermon.nodes.map((sermon, i) => (
         <div key={i}>
-          {sermon.Title} {sermon.VideoLink} {sermon.Series.Name}{" "}
-          {sermon.Preacher.Prefix} {sermon.Preacher.Name}
+          <SermonCard 
+             key={i}
+            title={sermon.Title}
+            date={sermon.DatePreached}
+            img={
+              <StaticImage
+                src="../../../images/Sermon-Ad-Selah-April.png"
+                alt="Selah"
+              />
+            }
+            speaker={sermon.Preacher.Prefix + " " + sermon.Preacher.Name}
+            passage={sermon.Title}
+            series={sermon.Series.Name}
+          />
         </div>
```

Comment:
remove div here.  By removing the div will make the Sermon Card have the same height in each row.  i notice that the height are not consistent with the outer divs. after that all should be good.